server/storage/backend/hooks_test.go

```diff
@@ -41,8 +41,6 @@ func TestBackendPreCommitHook(t *testing.T) {
 	// Empty commit.
 	tx.Commit()
 
-	write(tx, []byte("foo"), []byte("bar"))
```

Comment:
In case someone asks why I removed this line...  it causes instability, because the periodical commit goroutine may contribute one more commit during the test. The symptom is the test may occasionally fail, see error message below,
```

$ go test -run TestBackendPreCommitHook
--- FAIL: TestBackendPreCommitHook (0.29s)
    hooks_test.go:120: OnPreCommit executed: key >
    hooks_test.go:120: OnPreCommit executed: key >c
    hooks_test.go:120: OnPreCommit executed: key >cc
    hooks_test.go:46: 
        	Error Trace:	hooks_test.go:46
        	Error:      	Not equal: 
        	            	expected: ">cc"
        	            	actual  : ">ccc"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	->cc
        	            	+>ccc
        	Test:       	TestBackendPreCommitHook
        	Messages:   	expected 2 explict commits
    hooks_test.go:120: OnPreCommit executed: key >ccc
    hooks_test.go:48: 
        	Error Trace:	hooks_test.go:48
        	Error:      	Not equal: 
        	            	expected: ">ccc"
        	            	actual  : ">cccc"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	->ccc
        	            	+>cccc
        	Test:       	TestBackendPreCommitHook
        	Messages:   	expected 3 explict commits
    hooks_test.go:120: OnPreCommit executed: key >cccc
FAIL
exit status 1
FAIL	go.etcd.io/etcd/server/v3/storage/backend	0.750s
```